doc/modules/model_evaluation.rst

```diff
@@ -215,6 +215,22 @@ the following two rules:
   Again, by convention higher numbers are better, so if your scorer
   returns loss, that value should be negated.
 
+In case you require the use of parallelisation (using ``n_jobs`` > 1) with a 
+custom scoring function, please make sure to define the custom function in a 
+module and import it for usage. 
+
+For example, consider the following use of the custom scoring function ``rmse`` 
+in the below code.
+``cross_val_score(elnet_pipe,
```

Comment:
this is not Restructured text formatting

Reply:
@jnothman added both. Apologies for overlooking the formatting.

Reply:
@jnothman - just realised from a build error that I can't add an import statement without the module not being present physically. So just wanted to check, when you said this needs an example of importing, is this line `from custom_scorer_module import rmse` that you wanted me to add ?

Reply:
If you do it in a doctest section, you will need to flag it to be skipped, such as with ` # doctest: +SKIP`